src/validate/index.ts

```diff
@@ -426,6 +428,37 @@ class ValidateGenerator extends Generator {
         this.emit(validationErrorType.WARN, objectToEmit);
       });
   }
+
+  _validateExplorer(global: GlobalJson, orgs: OrgJson[]): void {
+    if (global.tools?.explorer === true) {
+      orgs
+        .filter((o) => o.tools?.explorer === true)
+        .forEach((o) => {
+          const objectToEmit = {
+            category: validationCategories.ORGS,
+            message: `Explorer for organization '${o.organization.name}' is enabled, however it will be ignored due to global explorer enabled.`,
+          };
+          this.emit(validationErrorType.WARN, objectToEmit);
+        });
+    }
+  }
+
+  _validateExplorerWithFabricVersion(global: GlobalJson, orgs: OrgJson[]): void {
+    const version: number[] = global.fabricVersion.split(".").map((n) => parseInt(n));
+
+    if ((version[0] === 1 && version[1] <= 3) || (version[0] === 2 && version[1] >= 4)) {
```

Comment:
mamy na to util `!version(fabricVersion).isGreaterOrEqual("1.4.0") || version(fabricVersion).isGreaterOrEqual("2.4.0") => ERR`